src/map/slice.rs

```diff
@@ -201,6 +201,59 @@ impl<K, V> Slice<K, V> {
     pub fn into_values(self: Box<Self>) -> IntoValues<K, V> {
         IntoValues::new(self.into_entries())
     }
+
+    /// Search over a sorted map for a value.
```

Comment:
```suggestion
    /// Search over a sorted map for a key.
```

(I'm tempted to say "key value", but "value" has a particular meaning in maps.)

Reply:
So does "key". Perhaps "item"? Let me see what I can do…

Reply:
In this particular case, you *are* looking at the map key.

Reply:
Sorry, I figured that out only after posting my comment.